test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Atomic Kickstart default.snap.txt

```diff
@@ -57,6 +57,8 @@ rm -f /etc/ostree/remotes.d/*.conf
 
 
 
+
```

Comment:
we should add the snippet with `-%>` to avoid extra empty lines

Reply:
`-%>` added but still having empty row in snaps.